packages/amplify-util-uibuilder/src/__tests__/syncAmplifyUiComponents.test.ts

```diff
@@ -94,14 +152,14 @@ describe('should sync amplify ui builder components', () => {
 
   it('does not throw an error when createUiBuilderComponent fails', () => {
     createUiBuilderComponentDependencyMock.createUiBuilderComponent = jest.fn(() => {
-      throw new Error('ahhh!');
+      throw new Error('ahhh!'); // eslint-disable-line spellcheck/spell-checker
```

Comment:
nit: feel free to leave it since is existing code, but should we just change the `ahhh` text to something the spellcheck is not complaining instead of dismissing the warning?